docker/tests/curl/Dockerfile

```diff
@@ -0,0 +1,26 @@
+ARG FROM_IMAGE
+FROM ${FROM_IMAGE} as prepare-curl
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Always tag the version of an image explicitly](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5442148)